elastic/kibana
# [Upgrade Assistant] Add missing cluster privilege check

Fixes: https://github.com/elastic/kibana/issues/178703

## Summary

This PR adds a permission check for `cluster:manage_security` in the deprecation logs section and into the kibana logs section in order to more gracefully handle the cases in which this is missing.

##### How to test

* Start up kibana and es
* Create a new Role that has the `manage` cluster privilege and full kibaba privileges
* Create a new user that has that newly created role and login with it
* Navigate to Stack management -> Upgrade Assistant
* Verify that:
  * There is a warning callout for a missing `manage_security` privilege
  * Add the privilege to the used role and verify that the callout is gone


<details>
<summary>Screenshots</summary>

// Cluster privilege missing
![Screenshot 2024-03-20 at 11 03 39](https://github.com/elastic/kibana/assets/1191206/5c351a12-108b-44fe-a9ae-1358f364bafb)

// Cluster and index privileges missing
![Screenshot 2024-03-20 at 11 14 04](https://github.com/elastic/kibana/assets/1191206/dbf6799a-10aa-4add-b4ca-4cc6670b7a27)

// Missing cluster privilege in kibana issues page
![Screenshot 2024-03-20 at 13 03 14](https://github.com/elastic/kibana/assets/1191206/349bc35d-9317-45a6-b926-adf588547288)


</details>
